recipes/cosmic_profiles/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "cosmic_profiles" %}
+{% set version = "1.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c4c69a068e81e60309e23c1a8e3ee453d92ec3340a334dff83d2b67e2a6635aa
+
+build:
+  script_env:
+    - OMPI_MCA_plm_rsh_agent=  # [linux]
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+  host:
+    - python
+    - pip
+    - setuptools
+    - cython
+    - cythongsl
+    - numpy >=1.19.2
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy
+    - matplotlib-base
+    - scikit-learn
+    - cython
+    - cythongsl
+    - mpi4py
+    - pathos
+    - h5py <3.2
+    - psutil
+    - openmpi =4.0.4  # [linux]
+    - mpich  # [osx]
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+
+test:
+  imports:
+    - cosmic_profiles
+  source_files:
+    - cosmic_profiles/tests
+  requires:
+    - pytest
+    - pip
+  commands:
+    - pip check
+    - pytest cosmic_profiles/tests  # [linux and osx]
```

Comment:
Can we skip the tests that are expected to fail on Windows? You can do that in the feedstock. You can use `-k "not ( name-of-the-test)"`

Reply:
@ocefpaf good to know. Many thanks for merging!